gamestonk_terminal/options/syncretism_view.py

```diff
@@ -203,6 +211,129 @@ def screener_output(other_args: List[str]):
         print(e, "\n")
 
 
+def check_presets(preset_dict: dict):
+    """Checks option screener preset values
+
+    Parameters
+    ----------
+    preset_dict: dict
+        Defined presets from configparser
+    Returns
+    -------
+    error: str
+        String of all errors accumulated
+    """
+    float_list = [
+        "min-diff",
+        "max-diff",
+        "min-ask-bid",
+        "max-ask-bid",
+        "min-exp",
+        "max-exp",
+        "min-price",
+        "max-price",
+        "min-price-20d",
+        "max-price-20d",
+        "min-volume-20d",
+        "max-volume-20d",
+        "min-iv-20d",
+        "max-iv-20d",
+        "min-delta-20d",
+        "max-delta-20d",
+        "min-gamma-20d",
+        "max-gamma-20d",
+        "min-theta-20d",
+        "max-theta-20d",
+        "min-vega-20d",
+        "max-vega-20d",
+        "min-rho-20d",
+        "max-rho-20d",
+        "min-price-100d",
+        "max-price-100d",
+        "min-volume-100d",
+        "max-volume-100d",
+        "min-iv-100d",
+        "max-iv-100d",
+        "min-delta-100d",
+        "max-delta-100d",
+        "min-gamma-100d",
+        "max-gamma-100d",
+        "min-theta-100d",
+        "max-theta-100d",
+        "min-vega-100d",
+        "max-vega-100d",
+        "min-rho-100d",
+        "max-rho-100d",
+        "min-sto",
+        "max-sto",
+        "min-yield",
+        "max-yield",
+        "min-myield",
+        "max-myield",
+        "min-delta",
+        "max-delta",
+        "min-gamma",
+        "max-gamma",
+        "min-theta",
+        "max-theta",
+        "min-vega",
+        "max-vega",
+        "min-cap",
+        "max-cap",
+    ]
+    bool_list = ["active", "stock", "etf", "puts", "calls", "itm", "otm", "exclude"]
+    error = ""
+    for key, value in preset_dict.items():
+        if key in float_list:
+            try:
+                float(value)
+                if value.startswith("."):
+                    error += f"{key} : {value} needs to be formatted with leading 0\n"
+            except Exception:
+                error += f"{key} : {value}, should be float\n"
+
+        elif key in bool_list:
+            if value not in ["true", "false"]:
```

Comment:
I would not rely on quotes, and try to use .eval to evaluate a string to a type. If it fails, we know the user input something wrongly.